src/os_dbd/db_op.c

```diff
@@ -173,7 +173,8 @@ void *mysql_osdb_connect(const char *host, const char *user, const char *pass, c
         merror(DBINIT_ERROR);
         return (NULL);
     }
-
+    unsigned int timeout= 5;
```

Comment:
Let's keep the default value and in any case, improve this in a separate issue.

Reply:
Changed!